src/pages/index.tsx

```diff
@@ -59,29 +62,38 @@ function Page() {
 
       <IncentivesHeader incentives={IncentivesMock} />
 
-      <ProductShelf
-        first={ITEMS_PER_SECTION}
-        selectedFacets={[{ key: 'productClusterIds', value: '140' }]}
-        title="Most Wanted"
-      />
+      <Suspense fallback={<ProductShelfSkeleton />}>
+        <ProductShelf
+          first={ITEMS_PER_SECTION}
+          selectedFacets={[{ key: 'productClusterIds', value: '140' }]}
+          title="Most Wanted"
+          suspense
+        />
+      </Suspense>
 
-      <ProductTiles
-        first={3}
-        selectedFacets={[{ key: 'productClusterIds', value: '141' }]}
-        title="Just Arrived"
-      />
+      <Suspense fallback={<ProductTilesSkeleton loading />}>
```

Comment:
Really minor, but for consistency's sake, noticed `loading` defaults to true, and in other uses of `ProductTilesSkeleton` (and `ProductShelfSkeleton`) on this same file, we're not explicitly passing `loading`, as we're here. 

Can we either add `loading` to the missing ones or remove them altogether (vote for this one)? 
```suggestion
      <Suspense fallback={<ProductTilesSkeleton />}>
```

I fear someone glancing over the code might look at this inconsistency and question if the skeletons not using `loading` are really "loading".

![CleanShot 2022-05-20 at 15 22 00](https://user-images.githubusercontent.com/381395/169589687-a21a6fbf-fe2f-4771-a3ee-b860e4f1b2d5.png)

Reply:
Fixed